lib/features/upload/domain/state/attachment_upload_state.dart

```diff
@@ -54,10 +54,7 @@ class SuccessAttachmentUploadState extends Success {
 class ErrorAttachmentUploadState extends FeatureFailure {
   final UploadTaskId uploadId;
 
-  ErrorAttachmentUploadState({
-    required this.uploadId,
-    dynamic exception
-  }) : super(exception: exception);
+  ErrorAttachmentUploadState({required this.uploadId});
```

Comment:
It should not be deleted. We may need to show toast for each exception

Reply:
Done